src/metaschema/oscal_profile_metaschema.xml

```diff
@@ -392,15 +411,5 @@
       </allowed-values>
    </define-flag>
    
-   <define-flag as-type="NCName" name="position">
-      <formal-name>Position</formal-name>
-      <description>Where to add the new content with respect to the targeted element (beside it or inside it)</description>
-      <allowed-values><!-- TODO: define these -->
-         <enum value="before"/>
-         <enum value="after"/>
-         <enum value="starting"/>
-         <enum value="ending"/>
-      </allowed-values>
-   </define-flag>
```

Comment:
@brianrufgsa Can you explain this change?

Reply:
@david-waltermire-nist I did all that, and when I prepared the pull request it only showed my eight files. I am not sure what is happening. It was only supposed to be the four FedRAMP baselines and resolved profiles.